libretroshare/src/retroshare/rswire.h

```diff
@@ -93,9 +95,10 @@ class RsWirePlace
 #define  WIRE_PULSE_TYPE_REPLY_MSG           (0x0002)
 #define  WIRE_PULSE_TYPE_REPLY_REFERENCE     (0x0004)
 
-#define  WIRE_PULSE_TYPE_SENTIMENT_POSITIVE  (0x0010)
-#define  WIRE_PULSE_TYPE_SENTIMENT_NEUTRAL   (0x0020)
-#define  WIRE_PULSE_TYPE_SENTIMENT_NEGATIVE  (0x0040)
+#define  WIRE_PULSE_SENTIMENT_NO_SENTIMENT   (0x0000)
```

Comment:
Now we preferably use 
enum class WireSentiment: uint8_t {
         NoSentiment = 0x00,
         Positive         = 0x01,
       [etc]
}
Then you use it using e.g. WireSentiment::Positive. It's not critical, but the big advantage is that it is type-safe.